src/priorityq.py

```diff
@@ -7,7 +7,7 @@ def __init__(self, iterable=None):
         '''Initialise a Priorityq from a list of lists or tuples
             where the zeroeth element is the priority'''
         if iterable is None:
-            self.priorityq = None
+            self.priorityq = []
         else:
             try:
                 self.priorityq = sorted(iterable, key=lambda tup: tup[0])
```

Comment:
Note: a list of tuples will be sorted by the first item in the tuple by default. Tuples compare to each other based on the first item, then on the second item, etc.